arkane/ess/molpro.py

```diff
@@ -34,8 +34,8 @@
 
 import logging
 import math
-
 import numpy as np
+import os.path
```

Comment:
Maybe too picky (**if so neglect this comment**). In most `.py` file in RMG, we put built-in modules before exterior modules.  For here, `os` is a built-in module while `numpy` is not; so better put `import os.path` before `import numpy ..` and add an extra empty line.

Reply:
Thanks for letting me know. I'll update this